From: Shengjing Zhu <zhsj@debian.org>
Date: Fri, 7 Feb 2020 23:29:11 +0800
Subject: Increase default test timeout to 30m

Forwarded: not-needed
---
 integration/integration_suite_test.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/integration/integration_suite_test.go b/integration/integration_suite_test.go
index 5b009a0..db586ad 100644
--- a/integration/integration_suite_test.go
+++ b/integration/integration_suite_test.go
@@ -31,7 +31,7 @@ func init() {
 }
 
 func TestIntegration(t *testing.T) {
-	SetDefaultEventuallyTimeout(30 * time.Second)
+	SetDefaultEventuallyTimeout(30 * time.Minute)
 	format.TruncatedDiff = false
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "Integration Suite", Label("integration"))
